Update installation instructions (#3712)
DataDog/integrations-core: cacti/README.md

## Setup
### Installation

The Cacti check is included in the [Datadog Agent][1] package, to start gathering metrics you first need to:

> - Install librrd headers and librairies

Reviewer: ```suggestion
- Install librrd headers and libraries
```

- Install python bindings to rrdtool

#### librrd headers and librairies

On Debian/Ubuntu
```shell
sudo apt-get install librrd-dev
```

On RHEL/CentOS
```shell
sudo yum install rrdtool-devel
```

#### Python bindinges

> Now add the `rrdtool` python package to the agent with the following command.

Reviewer 2: ```suggestion
Now add the `rrdtool` Python package to the Agent with the following command.
```

```shell

> sudo -u dd-agent /opt/datadog-agent/embedded/bin/pip install rrdtool

Reviewer 3: This library has a memory leak, which can have significant impact depending on the number of rrd files to read. See https://github.com/commx/python-rrdtool/issues/25

### Service Checks
The Cacti check does not include any service checks.

## Known issues

> The python library used by this integration appears to leak memory under certain circonstances. If you experience it, one workaround is to install the [`python-rrdtool`][6] package instead of `rrdtool`. This old package is not maintained anymore and is not officialy supported by this integration but some have had good results by doing that.

Reviewer: ```suggestion
The python library used by this integration appears to leak memory under certain circumstances. If you experience it, one workaround is to install the [`python-rrdtool`][6] package instead of `rrdtool`. This old package is not maintained anymore and is not officially supported by this integration but some have had good results by doing that.
```

> A [Github issue][7] has been opened to track this memory leak.

Reviewer 2: Move this section under `Troubleshooting` and update text as shown below.

> ## Troubleshooting

Reviewer 2: ### Known issues

 The Python library used by this integration leaks memory under certain circumstances. If you experience this, one workaround is to install the [`python-rrdtool`][6] package instead of `rrdtool`. This older package is not maintained and is not officially supported by this integration but it has help others resolve the memory issues.

 A [Github issue][7] has been opened to track this memory leak.

The Cacti check does not include any service checks.

## Troubleshooting
### Known issues

> The Python library used by this integration leaks memory under certain circumstances. If you experience this, one workaround is to install the [python-rrdtool][6] package instead of rrdtool. This older package is not maintained and is not officially supported by this integration but it has help others resolve the memory issues.

Reviewer: ```suggestion
The Python library used by this integration leaks memory under certain circumstances. If you experience this, one workaround is to install the [python-rrdtool][6] package instead of rrdtool. This older package is not maintained and is not officially supported by this integration but it has helped others resolve the memory issues.
```

Author: 👍
